testdata/cheats/Cheats.sol

```diff
@@ -103,4 +103,23 @@ interface Cheats {
     function startBroadcast(address) external;
     // Stops collecting onchain transactions
     function stopBroadcast() external;
+    // Reads the entire content of file to string, (path) => (data)
+    function readFile(string calldata) external returns (string memory);
+    // Reads next line of file to string, (path) => (line)
+    function readLine(string calldata) external returns (string memory);
```

Comment:
I've thought about how solidity `Fs` lib could look like:

https://github.com/mattsse/foundry/blob/e21fefd9488240e5f62378671be5da7691d6170b/evm-adapters/script-lib/Fs.sol

idea was to have a solidity struct that maps to a file and via using Interface you could call `std::fs::File` functions directly on it

having them all in the `VM` is fine though

Reply:
Feels like we can think of utilities built on top with forge std?